src/condor_schedd.V6/qmgmt.h

```diff
@@ -170,6 +172,8 @@ inline int USERRECID_to_qkey2(unsigned int userrec_id) {
 const int JOBSETID_qkey2 = -100;
 const int CLUSTERID_qkey2 = -1;
 const int CLUSTERPRIVATE_qkey2 = -2;
+const int OCU_qkey2 = -3;
```

Comment:
the way this id is used is not actually has a key in the job queue.  this is very confusing.   I don't think you need to reserve and id here, since this is only for the negotiator communication.  But if you do, a comment is needed to make it clear that this is reserving a key id for negotiator communication, not for the job queue.